test/integration/Elsa.Workflows.IntegrationTests/Scenarios/HttpRequestWithLiquid/HttpRequestWithLiquidWorkflow.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Elsa.Expressions.Models;
+using Elsa.Http;
+using Elsa.Workflows.Activities;
+using Elsa.Workflows.Contracts;
+using Elsa.Workflows.Memory;
+using Elsa.Workflows.Models;
+using Elsa.Workflows.Services;
+
+namespace Elsa.Workflows.IntegrationTests.Scenarios.LiquidLists;
+
+/// <summary>
+/// A workflow that use FlowSendHttpRequest to get some data, use them in a ForEach with some liquid expressions
+/// </summary>
+public class HttpRequestWithLiquidWorkflow : WorkflowBase
+{
+    protected override void Build(IWorkflowBuilder builder)
+    {
+        var products = new Variable<object> { Name = "Products", StorageDriverType = typeof(WorkflowStorageDriver) };
+        var product = new Variable<object> { Name = "Product", StorageDriverType = typeof(WorkflowStorageDriver) };
+
+        builder.Root = new Sequence
+        {
+            Variables = { products, product },
+            Activities =
+            {
+                new FlowSendHttpRequest
+                {
+                    Url = new Input<Uri?>(new Uri("https://fakestoreapi.com/products?limit=5")),
+                    ParsedContent = new Output<object?>(products)
+                },
+                new WriteLine(new Expression("Liquid", "First product id: {{ Variables.Products[0].id }}")),
+                new WriteLine(new Expression("Liquid", "First product price rounded: {{ Variables.Products[0].price | round }}")),
+                new WriteLine(new Expression("Liquid", "First product as json: {{ Variables.Products[0] | json }}")),
+                new WriteLine(new Expression("Liquid", "Second product id: {{ Variables.Products[1].id }}")),
+                new FlowSendHttpRequest
+                {
+                    Url = new Input<Uri?>(new Expression("Liquid", "https://fakestoreapi.com/products/{{ Variables.Products[1].id }}")),
```

Comment:
I am not sure it's such a good idea to make HTTP requests to external services during integration tests, as it might cause the test to fail in case the external site is not available, which is a dependency we shouldn't want here.

As an alternative, perhaps replace this activity with a RunJavaScript activity that produces the JSON payload as its output?

I can push that change if you want after merging the PR, just let me know.